inference_client/client.py

```diff
@@ -45,7 +59,13 @@ def get_model(
                 'Please provide an endpoint or a valid user token to access the model.'
             )
 
-        if model_name:
+        from urllib.parse import urlparse
+
+        # Note: if endpoint is provided, model_name is ignored
+        o = urlparse(endpoint or model_name)
+        if o.scheme and o.netloc:
+            endpoint = endpoint or model_name
+        elif model_name:
```

Comment:
We can go back to single `inference_model_name_or_endpoint` param 😮‍💨

Reply:
nope, I don't want to introduce new param.

Reply:
And what's more, we don't need to update our doc with this approach.